src/Workspaces/Core/Portable/Workspace/Solution/ProjectState.cs

```diff
@@ -25,7 +25,9 @@ internal partial class ProjectState
         private readonly IReadOnlyList<DocumentId> _additionalDocumentIds;
         private readonly AsyncLazy<VersionStamp> _lazyLatestDocumentVersion;
         private readonly AsyncLazy<VersionStamp> _lazyLatestDocumentTopLevelChangeVersion;
-        private AnalyzerOptions _analyzerOptions;
+
+        // this will be initialized lazily.
```

Comment:
Why not just use Lazy<>? Also, should this be XML doc comments so it can point to where it's initialized?